openedx/core/djangoapps/catalog/utils.py

```diff
@@ -109,15 +109,17 @@ def get_program_types(name=None):
         return []
 
 
-def get_programs_with_type(types=None):
+def get_programs_with_type(types=None, include_hidden=True):
```

Comment:
Is this really where you want to be doing this filtering? Is it okay for hidden programs to show up on learners' dashboards? Should learners be able to earn certificates for hidden programs?